src/back/Update/HighPriority.php

```diff
@@ -195,8 +198,12 @@ private function processSubsectionTopics(array $topics, callable $avgProcessor):
 
             // Поиск нужных данных о новых раздачах.
             if (count($topicsInsert)) {
-                // Получить описание новых раздач.
-                $response = $this->apiClient->getTopicsDetails(array_keys($topicsInsert));
+                $this->logger->notice(
+                    'Получаем дополнительные сведения для новых раздач ({count} шт.) через API...',
```

Comment:
This note will be performed in a cycle for 18284 topics, for groups of 500. Are you sure that 40 notice is a good idea? What is the purpose of this information and why is it in the logs?

Reply:
I agree with your objection; moreover, this line never appeared in my logs, doesn't seem typical for my workflows.